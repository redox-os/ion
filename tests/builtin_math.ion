echo '# ANCHOR: math_non_interactive'
math -6*7
math - 50 8
echo '# ANCHOR_END: math_non_interactive'
echo '# ANCHOR: math_interactive'
echo $(math) | sed -n '1p'
echo '# ANCHOR_END: math_interactive'
echo '# ANCHOR: math_interactive_quiet'
echo $(math -q)
echo '# ANCHOR_END: math_interactive_quiet'
