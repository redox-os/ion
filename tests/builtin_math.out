# ANCHOR: math_non_interactive
-42
42
# ANCHOR_END: math_non_interactive
# ANCHOR: math_interactive
Ion's integrated calculator
# ANCHOR_END: math_interactive
# ANCHOR: math_interactive_quiet

# ANCHOR_END: math_interactive_quiet
